peps/pep-0788.rst

```diff
@@ -421,43 +417,43 @@ Strong Interpreter References
 .. c:type:: PyInterpreterRef
 
    An opaque, strong reference to an interpreter.
+
    The interpreter will wait until a strong reference has been released
    before shutting down.
 
    This type is guaranteed to be pointer-sized.
 
-.. c:function:: int PyInterpreterRef_Get(PyInterpreterRef *ref)
+.. c:function:: PyInterpreterRef PyInterpreterRef_FromCurrent(void)
```

Comment:
I prefer `PyInterpreterRef_Get()` and `PyInterpreterWeakRef_Get()` which are consistent with existing `PyThreadState_Get()` and `PyInterpreterState_Get()`.